Guesser/AbstractTypeGuesser.php

```diff
@@ -11,9 +11,9 @@
 
 namespace Sonata\PropelAdminBundle\Guesser;
 
+use Propel\Bundle\PropelBundle\Form\PropelTypeGuesser;
 use Sonata\AdminBundle\Guesser\TypeGuesserInterface;
 use Sonata\AdminBundle\Model\ModelManagerInterface;
-use Symfony\Bridge\Propel1\Form\PropelTypeGuesser;
```

Comment:
If you know nothing about Propel why are you a contributor on this porject? I use this bundle in production since 2 years.